docs/docsite/rst/dev_guide/platforms/vmware_guidelines.rst

```diff
@@ -181,6 +181,24 @@ Depending upon the functionality provided by ESXi or vCenter, some modules can s
 Functional tests
 ----------------
 
+Writing new tests
+~~~~~~~~~~~~~~~~~
+
+If you are writing a new collection of integration tests, there are a few VMware-specific things to note beyond
+the standard Ansible :ref:`integration testing<testing_integration>` process.
+
+The test-suite uses a set of common, pre-defined vars located in the :file:`test/integration/targets/prepare_vmware_tests/` role.
+The resources defined there are automatically created by importing that role are the start of your test:
+
+.. code-block:: yaml
+
+  - import_role:
+      name: prepare_vmware_tests
+    vars:
+      setup_datacenter: true
+
+This will give you a ready to use cluster, datacenter, datastores, folder, switch, dvswitch, ESXi hosts, and VMs.
```

Comment:
Can we mention the name of the each parameter in doc? ex: datacenter: {{dc1}}, cluster: {{ccr1}}.

Reply:
My inclination is to just link to the role vars in the repo, and not duplicate them here as there would be a risk of them getting out of sync if we update the role. What do you think @goneri?  I don't think the vars will change often, but at the same time the less often we touch them we might forget that we also need to update these docs.

Reply:
I agree.. thanks for clarification.

Reply:
I don't have a strong opinion on this. The variable names are a bit confusing IMO (e.g: `ccs1` should  be `cluster1`), this for historical reason. I would prefer to fix that before.

In the meantime, I suggest we just point the user to https://github.com/ansible/ansible/tree/602aded6ecfdb20197bc6cfbc7b08ca01f5a7814/test/integration/targets/prepare_vmware_tests/vars